test/Interop/C/stdlib/Inputs/include-signal.h

```diff
@@ -0,0 +1,3 @@
+#include <signal.h>
```

Comment:
It would be good to also have a swift test that imports `Glibc` with `--enable-cxx-interop`.